lib/engines/content_object_store/app/components/content_object_store/content_block_edition/show/confirm_summary_list_component.rb

```diff
@@ -0,0 +1,36 @@
+class ContentObjectStore::ContentBlockEdition::Show::ConfirmSummaryListComponent < ViewComponent::Base
+  def initialize(content_block_edition:)
+    @content_block_edition = content_block_edition
+  end
+
+private
+
+  attr_reader :content_block_edition
+
+  def items
+    details_items.push(confirm_item).push(date_item)
+  end
+
+  def details_items
+    content_block_edition.details.map do |key, value|
+      {
+        field: "New #{key.humanize.downcase}",
+        value:,
+      }
+    end
+  end
+
+  def confirm_item
+    {
+      field: "Confirm",
+      value: "I confirm that I am happy for the email to be changed on these pages.",
```

Comment:
need to programmatically add in block type rather than hard code `email`

Reply:
Maybe "content block" for now?